lib/Flux/SessionData.php

```diff
@@ -263,43 +263,45 @@ public function login($server, $username, $password, $securityCode = null)
 		if (!$loginAthenaGroup) {
 			throw new Flux_LoginError('Invalid server.', Flux_LoginError::INVALID_SERVER);
 		}
-		
+
 		if ($loginAthenaGroup->loginServer->isIpBanned() && !Flux::config('AllowIpBanLogin')) {
 			throw new Flux_LoginError('IP address is banned', Flux_LoginError::IPBANNED);
 		}
-		
+
 		if ($securityCode !== false && Flux::config('UseLoginCaptcha')) {
 			if (strtolower($securityCode) != strtolower($this->securityCode)) {
 				throw new Flux_LoginError('Invalid security code', Flux_LoginError::INVALID_SECURITY_CODE);
 			}
 			elseif (Flux::config('EnableReCaptcha')) {
 				require_once 'recaptcha/recaptchalib.php';
-				$resp = recaptcha_check_answer(
-					Flux::config('ReCaptchaPrivateKey'),
-					$_SERVER['REMOTE_ADDR'],
-					// Checks POST fields.
-					$_POST['recaptcha_challenge_field'],
-					$_POST['recaptcha_response_field']);
-				
-				if (!$resp->is_valid) {
+				$response;
```

Comment:
Same as above, does this have a purpose, or is it intended to be `$response = false;` or something similar?